bin/build-locales

```diff
@@ -50,15 +50,15 @@ poFiles.forEach((pofile) => {
     var defineLocale = null;
     try {
       const momentLocale = locale.replace('_', '-').toLowerCase();
+      // Check for the locale first; if it doesn't exist we don't have
+      // a moment locale that matches.
+      fs.accessSync(`./node_modules/moment/locale/${momentLocale}.js`);
+
       // We're using `new Function()` here to create a function out of the
-      // raw code in this file; this function won't be executed but will be
-      // written out by `toSource()` so that it can be used later (at runtime,
-      // by moment).
+      // raw code; this function won't be executed but will be written out by
+      // `toSource()` so that it can be used later (at runtime, by moment).
       defineLocale = new Function(
-        fs.readFileSync(
-          `./node_modules/moment/locale/${momentLocale}.js`, 'utf8'
-        )
-      );
+        `require('moment/locale/${momentLocale}.js');`);
```

Comment:
Since requiring this module is magical, I think it would make sense to add a comment to the auto-generated code like:

    // By requiring this module, the new locale is defined and registered internally for moment.